template-for-repository/proofs/Makefile.common

```diff
@@ -399,17 +529,41 @@ $(LOGDIR)/source.json:
 	$(MAKE_SOURCE) --srcdir $(SRCDIR) --wkdir $(PROOFDIR) > $@
 	$(RM) -r $(GOTODIR)
 
+define VIEWER2_CMD
+  $(VIEWER2) \
+    --result $(LOGDIR)/result.xml \
+    --coverage $(LOGDIR)/coverage.xml \
+    --property $(LOGDIR)/property.xml \
+    --srcdir $(SRCDIR) \
+    --goto $(HARNESS_GOTO).goto \
+    --reportdir $(PROOFDIR)/report
+endef
```

Comment:
Consider including a VIEWER2FLAGS variable here.

Reply:
As above, I would like to restrict this PR to purely porting to Litani.